src/Tablebot/Plugin/Discord.hs

```diff
@@ -35,6 +38,30 @@ sendMessage ::
   DatabaseDiscord (Either RestCallErrorCode Message)
 sendMessage m t = lift . restCall $ R.CreateMessage (messageChannel m) t
 
+-- | @sendEmbedMessage@ sends the input message @t@ in the same channel as message
+-- @m@ with an additional CreateEmbed. This returns an @Either RestCallErrorCode Message@ to denote failure or
+-- return the 'Message' that was just sent.
+sendEmbedMessage ::
+  Message ->
+  CreateEmbed ->
+  Text ->
+  DatabaseDiscord (Either RestCallErrorCode Message)
+sendEmbedMessage m e t = lift . restCall $ R.CreateMessageEmbed (messageChannel m) t e
+
+-- | @sendColouredEmbedMessage@ sends the input message @t@ in the same channel as message
+-- @m@ with an additional full Embed. This returns an @Either RestCallErrorCode Message@ to denote failure or
+-- return the 'Message' that was just sent.
+-- This is *really* janky. The library exposes *no way* to create a coloured embed through its main api,
+-- so I'm having to manually reimplement the sending logic just to add this in.
+-- If you suffer from nightmares, don't look in 'Tablebot.Handler.Embed'. Nothing good lives there.
+-- In the future, I may actually submit a PR to discord-haskell with a fix to allow colours properly.
+sendColouredEmbedMessage ::
```

Comment:
This has a different structure to `sendEmbedMessage`:

`Message -> CreateEmbed -> Text -> DatabaseDiscord (...)`
vs
`Message -> Text -> Embed -> DatabaseDiscord (...)`